source/elements/oneMKL/source/domains/dft/descriptor.rst

```diff
@@ -104,6 +114,8 @@ factors. The function :ref:`onemkl_dft_descriptor_commit` does this work
 after use of the function :ref:`onemkl_dft_descriptor_set_value` to set values 
 of all necessary parameters.
 
+The copy constructor copies by value.
```

Comment:
Personally I would understand "copy by value" as a shallow copy, meaning any pointers/references are shared. I think we want the copy constructor to create a deep copy.

Reply:
Using "deep-copy" in [9f90533](https://github.com/oneapi-src/oneAPI-spec/pull/490/commits/9f9053386a24e589f26254ba79f7157a693803aa).